source/hardware/controllers/denon_mc7000.rst

```diff
@@ -77,326 +50,216 @@ thread.
+3.        :hwlabel:`LEVEL` knob                                            Turn to adjust the level of the pre-fader, pre-EQ audio signal for the channel.
+4.        Channel Level Meters                                             Display the audio signal level of the channel as in Mixxx GUI.
+5.        Channel EQ knobs                                                 Turn to boost or cut the :hwlabel:`HI`, :hwlabel:`MID` and :hwlabel:`LOW` frequencies for the channel.
+6.        :hwlabel:`FILTER`                                                Turn this knob to adjust the filter applied to the channel. Turn the knob counterclockwise to apply a low-pass filter. Turn the knob clockwise to apply a high-pass Filter.
+7.        :hwlabel:`CUE` button                                            Press to send the channel’s pre-fader signal to the headphones’ cue channel.
+8.        Channel fader                                                    Use to adjust the channel’s volume level.
+9.        Crossfader Assign                                                Routes the audio playing on the corresponding channel to either side of the crossfader (:hwlabel:`A` or :hwlabel:`B`), or bypasses the crossfader and sends the audio directly to the program mix (:hwlabel:`THRU`).
+10.       Crossfader                                                       Use this crossfader to mix between the left and right decks.
+12.       :hwlabel:`SAMPLER LEVEL`                                         Turn this knob to adjust the volume level of the software’s sampler. If the volume is set to 0 then the Sampler banks are hidden, otherwise shown.
+13.       Master Level Meters                                              These LEDs show the Master Audio signal.
+14.       :hwlabel:`MASTER LEVEL` knob                                     Turn to adjust the volume level of the master outputs.
+15.       :hwlabel:`BOOTH LEVEL` knob                                      Turn to adjust the volume level of the booth outputs.
+20.       :hwlabel:`PHONES LEVEL` knob                                     Turn to adjust the volume level of the headphones.
+21.       :hwlabel:`PHONES MIX` knob                                       Adjusts the software’s audio output to the headphones, mixing between the cue output and the master mix output.
+22.       :hwlabel:`SPLIT` switch                                          When this switch is in the :hwlabel:`ON` position, the headphone audio will be “split” such that all channels sent to cue channel are summed to mono and sent to the left headphone channel and the master mix is summed to mono and sent to the right channel. When the switch is in the Off position, the cue channel and master mix will be “blended” together.
+          :hwlabel:`X FADER CONTOUR`                                       Turn to ajust the slope of the crossfader curve, to the left for a smooth fade (mixing) or to the right for a sharp cut (scratching.
```

Comment:
```suggestion
          :hwlabel:`X FADER CONTOUR`                                       Turn to adjust the slope of the crossfader curve, to the left for a smooth fade (mixing) or to the right for a sharp cut (scratching.
```

Reply:
I recognized that but also another spelling issue. Don't worry about that. Cheers